lib/string_tools.rb

```diff
@@ -235,7 +235,7 @@ def call(env)
 
         return unless node.name == 'iframe'
 
-        unless node[:src] =~ %r{^(http|https):?\/\/(www\.)?youtube?\.com\/}
+        unless node[:src] =~ %r{^http(?:s?):\/\/?(:www\.)?rutube\.ru(\/video|\/play\/embed)|youtu(?:be\.com\/(watch|shorts|embed)|\.be\/|be\-nocookie)}
```

Comment:
[Correctable] Style/RedundantRegexpEscape: Redundant escape inside regexp literal<br>Layout/LineLength: Line is too long. [151/120]